tee_supplicant/src/tee_supplicant.c

```diff
@@ -120,6 +131,519 @@ static int load_ta(uint32_t num_params, struct tee_param *params)
 	return TEEC_SUCCESS;
 }
 
+static int tee_fs_open(size_t num_params, struct tee_param *params, int flags)
```

Comment:
I guess all FS operations should be moved to separate file

Reply:
moved